CODEOWNERS

```
@@ -14,10 +14,10 @@
 /src/_sass/ @khanhnwin @parlough @sfshaza2 @atsansone @domesticmouse @MaryaBelanger
 
 # Covers website tooling
-/tool/      @atsansone @parlough @khanhnwin @domesticmouse @MaryaBelanger
+/tool/      @sfshaza2 @atsansone @parlough @khanhnwin @domesticmouse @MaryaBelanger
 
 # Covers examples in files
-/examples/  @domesticmouse @johnpryan @khanhnwin @parlough @MaryaBelanger @atsansone
+/examples/  @sfshaza2 @domesticmouse @johnpryan @khanhnwin @parlough @MaryaBelanger @atsansone
 
 # Covers all configuration files for the website
-/*          @atsansone @parlough @sfshaza2 @khanhnwin @domesticmouse @MaryaBelanger
+/*          @sfshaza2 @atsansone @parlough @sfshaza2 @khanhnwin @domesticmouse @MaryaBelanger
```

I think I'm seeing double 👀

```suggestion
/*          @sfshaza2 @atsansone @parlough @khanhnwin @domesticmouse @MaryaBelanger
```